test/setup/subchain_setup

```diff
@@ -0,0 +1,12 @@
+// Replace in /mangata-node/primitives/core/src/registry.rs
+
+static APP_REGISTRY: &[Entry] = &[
+	Entry {
+		app: App::ETH,
+		id: [252, 151, 166, 25, 125, 201, 11, 239, 107, 190, 253, 103, 39, 66, 237, 117, 233, 118, 133, 83],
```

Comment:
Are these IDs for Kovan or for a local ethereum chain?

Reply:
These are for the local ethereum chain. All addresses are local testnet addresses.